### PR TITLE
Common/header

### PR DESCRIPTION
**Pull request by MieczyslawFrontDev**
Hi friends! 

I've created a Header component in common folder beacause this component will be used on every page.
Also I,ve exported icons in svg format, installed normalize and font family poppins added.
I,ve implemented the header component for the mobile version with the breakpoinst.

### CODE REVIEW

File: src/common/Header/styled.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+`;
+
+export const Name = styled.a`
+  font-size: 14px;
+  padding: 8px 24px;
+  text-transform: uppercase;
+
+  &:hover {
+    border: 1px solid white;
+    margin: -1px;
```
Comment (paulgrym) [inline src/common/Header/styled.js:107]: Maybe changing "border" to "outline" would eliminate need to set margin to -1?

File: src/common/Header/styled.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+export const Link = styled.a`
+  display: flex;
+  align-items: center;
+  font-size: 24px;
+  font-weight: 500;
+  cursor: pointer;
+  letter-spacing: -1.5px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
```
Comment (paulgrym) [inline src/common/Header/styled.js:57]: letter-spacing for mobile should be  -0.5px

File: src/common/Header/styled.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  cursor: pointer;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    margin-right: 12px;
+  }
+`;
+
+export const Name = styled.a`
+  font-size: 14px;
+  padding: 8px 24px;
```
Comment (paulgrym) [inline src/common/Header/styled.js:102]: I think top and bottom padding are to small - border width to border height ratio seems to differ from the design

File: src/common/Header/styled.js (hunk 1 above)
Comment (MieczyslawFrontDev) [reply src/common/Header/styled.js:107]: Thanks for the suggestion. I checked it and it's a better solution 👍

File: src/common/Header/styled.js (hunk 2 above)
Comment (MieczyslawFrontDev) [reply src/common/Header/styled.js:57]: Ok. I will correct it. Thanks 😄

File: src/common/Header/styled.js [hunk 4]
```diff
@@ -0,0 +1,154 @@
+import styled from "styled-components";
+import { ReactComponent as VideoSVG } from "../Header/Icons/video.svg";
+import { ReactComponent as SearchSVG } from "../Header/Icons/search.svg";
+
+export const Wrapper = styled.header`
+  max-width: 1920px;
+  margin: 0 auto;
+  background-color: #18181b;
+  color: white;
+  margin-bottom: 58px;
```
[the remark this answers is not in the document]
Comment (MieczyslawFrontDev) [reply src/common/Header/styled.js:10]: 👍

File: src/common/Header/styled.js (hunk 3 above)
Comment (MieczyslawFrontDev) [reply src/common/Header/styled.js:102]: I checked this and You're right. Top/bottom padding should be 13.5px. Thanks

**Pull request merged**